src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java

```diff
@@ -128,9 +128,9 @@ public void onMessage(Message message) {
                             IngestReport errorReport = new IngestReport();
                             errorReport.setFailure();
                             if (ex.getMessage() != null) {
-                                errorReport.setReport("Ingest succeeded, but failed to save the ingested tabular data in the database: " + ex.getMessage());
+                                errorReport.setReport(BundleUtil.getStringFromBundle("file.ingest.saveFailed.detail.message") + ex.getMessage());
```

Comment:
To artificially test this I used the following code...

```
     public boolean ingestAsTabular(Long datafile_id) {
+        if (true) {
+            throw (new RuntimeException("customMessage"));
+        }
         DataFile dataFile = fileService.find(datafile_id);
```

... to create this screenshot, which looks good to me:

![Screen Shot 2023-05-26 at 2 47 48 PM](https://github.com/IQSS/dataverse/assets/21006/3cfbd603-aa0c-4819-b666-0be603451af9)